DataLayer/MpptData/I_MpptData.h

```diff
@@ -10,16 +10,10 @@ class I_MpptData : public QObject
 public:
     virtual ~I_MpptData() {}
 
-    virtual Mppt mpptZero() const = 0;
-    virtual Mppt mpptOne() const = 0;
-    virtual Mppt mpptTwo() const = 0;
+    virtual Mppt getMppt(int i) const = 0;
```

Comment:
Reiterating from facebook, I don't think we're using this anywhere so lets drop it for now.